src/Template/Pages/downloads.ctp

```diff
@@ -247,6 +250,34 @@ $transcriptionsOptions = $this->Downloads->createOptions('transcriptions');
             </dl>
         </div>
 
+        <!-- Sentences based on id  -->
+        <div  class="section md-whiteframe-1dp">
+            <?php /* @translators: section title in the Downloads page */ ?>
+            <h2><?= __('Sentences Based On ID') ?></h2>
+            <dl>
+                <dt><?= $filename ?></dt>
+                <?= $this->element(
+                    'per_language_files',
+                    [
+                        'model' => 'sentences',
+                        'options' => $sentencesBasedOnIdOptions
+                    ]
+                ) ?>
```

Comment:
The `per_language_files` element is meant to be used for export files that are split by languages. For example, the file `sentences.csv` is further proceeded by the [export script](https://github.com/Tatoeba/tatoeba2/blob/dev/docs/cron/export.sh) to produce multiple files inside the [per_language directory](https://downloads.tatoeba.org/exports/per_language/). `$this->element('per_language_files', ...)` allows to download these individual files with a dropdown box.

Since your export is only consisting of a single file `sentences_based_on_id.tar.bz2`, you should only display a single link to it. You can get inspiration from the way `links.tar.bz2` is displayed on the Downloads page.